src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/ResourceRepository.kt

```diff
@@ -87,14 +89,41 @@ class ResourceRepository(private val database: AppDatabase) : IResourceRepositor
             .map(this::buildResourceInfo)
     }
 
-    override fun getResources(collection: Collection, resourceInfo: ResourceInfo): Observable<Content> {
+    /**
+     * Returns all resources for which the resource content's COLLECTION_FK field references this collection.
+     * This will return resources about the chapter as well as all resources about the chapter's chunks.
+     */
+    override fun getResourcesForCollectionAndChildren(
```

Comment:
see note in the corresponding common PR about this [#100](https://github.com/WycliffeAssociates/otter-common/pull/100)

Reply:
The only reason I can think that we might want it is if we want to load all of the resources at once for a chapter to calculate progress. But we haven't discussed how we'll do that yet, and this function wouldn't be totally necessary anyway. So I'll remove it.